what_is.md

```diff
@@ -187,8 +195,8 @@ In fact, succinct tree sequences don't store each tree separately, but instead a
 based on an interconnected *genetic genealogy*, in which
 [genetic recombination](https://en.wikipedia.org/wiki/Genetic_recombination) has led
 to different regions of the chromosome having different histories. Another way of
-thinking about the tree sequence above is that it describes the full genetic
-*family "tree"* (strictly, "network") of our 10 genomes.
```

Comment:
I think it's helpful to mention "network" somewhere, as that's in the title. I also think a "genetic family tree" is worth mentioning somewhere, but perhaps not here. I'm happy to merge this and add something like this sentence in again at the top.